zirving/STEPCapstone
# Travel object component

-  Travel object component will render different sub components depending on the type property passed in props
- Has delete and edit buttons which are on the right side of the container
- Remainder of container will be filled by the content (event, flight, hotel)
- Created components folder for all our components and a styles folder for any additional css we may need.

To test:
In App.js load the TravelObject component in the App class:
< TravelObject type="event"  / >

Change the type to either event, flight, or hotel to see how it works.

When the individual components are ready, alter the TravelObject.js file to set content to your component to test its appearance. 

The component will span the entire screen right now because the size of the travel object will depend on the size of the itinerary.
